tests/IIM/flow_past_cylinder.mpirun=2.output

```diff
@@ -0,0 +1 @@
+../../../source/IBAMR/tests/IIM/flow_past_cylinder.mpirun=2.output
```

Comment:
Adding these things as symbolic links in this manner isn't going to work - you need to actually add the files here. The test runner will take care of setting things up appropriately in the build directory.